app/views/ops/_settings_advanced_tab.html.haml

```diff
@@ -6,7 +6,10 @@
         %span.pficon.pficon-error-octagon
         %span.pficon.pficon-error-exclamation
       %strong
-        = _("Caution: Manual changes to configuration files can disable the Server!")
+        - warning_msg = _("Caution: Manual changes to configuration files can disable the #{selected_node_title}!")
+        - if selected?(x_node, "z") || selected?(x_node, "svr")
```

Comment:
Minor, but technically this logic is duplicated here and in the selected_node_parent_title.  You could de-dup that logic by changing selected_node_parent_title to return nil for regions, and then there just do `if selected_node_parent_title`